src/pages/Dashboard/components/SummaryChart.tsx

```diff
@@ -24,15 +24,17 @@ interface Country {
 interface SummaryChartProps {
   data: Country[];
   filter: "sex" | "lgbtq";
+  w: number
+  h: number
 }
 
-export const SummaryChart = ({ data, filter }: SummaryChartProps) => {
+export const SummaryChart = ({ data, filter, w, h }: SummaryChartProps) => {
```

Comment:
instead of passing these as props, I would make two constants outside of the component in this file to define the width and height since they are never changing. you could add something like this in a line above the interface definition
`CHART_WIDTH = 900;`
`CHART_HEIGHT = 600;`

Reply:
I might just be missing something, but how would we then make a chart of a different size for the home page if the width/height are constants?

Reply:
I was thinking the chart on the homepage would just be a screenshot image of the default values originally. but now that I think about it more it should definitely be dynamic so that it updates if we change the default settings of the graph too.

Reply:
Looks good then :)

Reply:
Ahhh, I didn't consider the idea of a screenshot